src/api-utils.jsx

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+export function getArticles() {
+  return axios
+    .get("https://milos-nc-news.onrender.com/api/articles")
+    .then(({ data }) => {
+      return data.articles;
+    });
```

Comment:
much better! Have a look at axios.create() so you don't have to keep reusing the big URL for each request you make (you don't need to submit another PR though!) 😊